build.gradle

```diff
@@ -28,13 +28,11 @@ dependencies {
     implementation (
              'org.apache.httpcomponents:httpclient:4.5.8',
              'com.google.code.gson:gson:2.8.5',
-             'com.opera:operadriver:1.5',
-             'com.codeborne:phantomjsdriver:1.4.3',
-             'org.seleniumhq.selenium:selenium-server:3.141.59',
-             'org.seleniumhq.selenium:htmlunit-driver:2.36.0',
+             'org.seleniumhq.selenium:selenium-java:4.7.2',
+             'org.seleniumhq.selenium:htmlunit-driver:4.7.2',
              'net.htmlparser.jericho:jericho-html:3.1')
 
-    testImplementation("com.github.tomakehurst:wiremock-jre8:2.33.2") {
+    testImplementation("com.github.tomakehurst:wiremock-jre8:2.35.0") {
```

Comment:
Is this really targeting jre8? Is this something we should be changing?

Reply:
It's built with Java 8 version, the other artifact available is built with Java 7. The 3.x version will use Java 11 but that's still beta.

Reply:
Okay